modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/support/Support.kt

```diff
@@ -270,6 +272,13 @@ class Support @Inject constructor(
                 }
             }
 
+            runCatching {
```

Comment:
Do we need to catch the exception here? There is a try catch around the whole block.